examples/AdvancedOperations/AddAppCampaign.php

```diff
@@ -0,0 +1,443 @@
+
+        return $createdCampaignResourceName;
+    }
+
+    /**
+     * Sets campaign targeting criteria for a given campaign.
+     *
+     * Both location and language targeting are illustrated.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param string $campaignResourceName the resource name of the campaign to apply targeting to
+     */
+    private static function setCampaignTargetingCriteria(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $campaignResourceName
+    ) {
+        $campaignCriterionOperations = [];
+
+        // Creates the location campaign criteria.
+        // Besides using location_id, you can also search by location names from
+        // GeoTargetConstantService.suggestGeoTargetConstants() and directly
+        // apply GeoTargetConstant.resource_name here. An example can be found
+        // in GetGeoTargetConstantByNames.php.
+        $locationIds = [
+            21137, // California
+            2484   // Mexico
+        ];
+        foreach ($locationIds as $locationId) {
+            // Creates a campaign criterion.
+            $campaignCriterion = new CampaignCriterion([
+                'campaign' => new StringValue(['value' => $campaignResourceName]),
+                'type' => CriterionType::LOCATION,
+                'location' => new LocationInfo([
+                    'geo_target_constant' => new StringValue([
+                        'value' => ResourceNames::forGeoTargetConstant($locationId)
+                    ])
+                ])
+            ]);
+
+            // Creates a campaign criterion operation.
+            $campaignCriterionOperation = new CampaignCriterionOperation();
+            $campaignCriterionOperation->setCreate($campaignCriterion);
+
+            $campaignCriterionOperations[] = $campaignCriterionOperation;
+        }
+
+        // Creates the language campaign criteria.
+        $languageIds = [
+            1000, // English
+            1003  // Spanish
+        ];
+        foreach ($languageIds as $languageId) {
+            // Creates a campaign criterion.
+            $campaignCriterion = new CampaignCriterion([
+                'campaign' => new StringValue(['value' => $campaignResourceName]),
+                'type' => CriterionType::LANGUAGE,
+                'language' => new LanguageInfo([
+                    'language_constant' => new StringValue(['value' =>
+                        ResourceNames::forLanguageConstant($languageId)
+                    ])
+                ])
+            ]);
+
+            // Creates a campaign criterion operation.
+            $campaignCriterionOperation = new CampaignCriterionOperation();
+            $campaignCriterionOperation->setCreate($campaignCriterion);
+
+            $campaignCriterionOperations[] = $campaignCriterionOperation;
+        }
+
+        // Submits the criteria operations.
```

Comment:
Let's make it consistent with other functions in this example, e.g., add a bit more comments like "and print their information."

We probably don't need to make it consistent across files. But at least for this one, let's make all comments as consistent as possible.